programs/client/BuzzHouse/Utils/Hugeint.cpp

```diff
@@ -0,0 +1,601 @@
+#include <Hugeint.h>
+#include <UHugeint.h>
```

Comment:
These files should be called `HugeInt.h/cpp` and `UHugeInt.h/cpp`. Notice the uppercase `I`